deploy/kubernetes/helm/che/values.yaml

```diff
@@ -37,13 +37,15 @@ global:
   ingressAnnotationsPrefix: "nginx."
   # options: default-host, single-host, multi-host
   serverStrategy: multi-host
-  tls:
-    enabled: false
-    useCertManager: true
-    useStaging: true
-    secretName: che-tls
-    useSelfSignedCerts: false
-    email: user@example.com
+  # Allows to enable and configure TLS
```

Comment:
if you specify `--set global.tls.secretName='che-tls' it will enable tls in helm charts ?

Reply:
Good question. I'll check.

But honestly, now I'm not sure that it's really a good idea to provide commented section in values. 
IMHO From `values.yaml` file structure it looks nice, but:
1. best practices recommend avoiding nested values [1].
2. Assume that `--set global.tls.secretName='che-tls'` works fine, then previously to enable `tls` you should specify `--set global.tls.enabled=true` and all default values will be in a place. With new-approach, every value should be specified manually. Or the corresponding values file should be provided.
In the case of `secretName` value, it 'sa good thing, because it's more obvious that user needs to prepare this tls secret for Che Server. Probably I need to read more about helm charts, and maybe required values there.

I really would like to hear other options.

[1] https://helm.sh/docs/chart_best_practices/#flat-or-nested-values

Reply:
Reverted these changes